.gitignore

```diff
@@ -34,4 +35,5 @@
 /yapf.egg-info
 
 /.idea
+.vscode/settings.json
```

Comment:
Not important but here^ I would personally expect (a) `/.vscode/settings.json` with a trailing slash or just `/.vscode/` also with trialing slash here. Is status quo intended?

---

Apart from that, the parts of the pull request that I do understand look pretty good to me, this will be great gift to the yapf community :+1:

Reply:
Used `/.vscode/settings.json`, excluding all of `/.vscode/` prevents adding things like launch.json, extensions.json.

Reply:
Thanks!